src/uct/tcp/tcp_ep.c

```diff
@@ -562,8 +572,9 @@ static unsigned uct_tcp_ep_progress_data_tx(uct_tcp_ep_t *ep)
         }
     }
 
-    uct_pending_queue_dispatch(priv, &ep->pending_q,
-                               uct_tcp_ep_ctx_buf_empty(&ep->tx));
+    if (!ucs_queue_is_empty(&ep->pending_q)) {
```

Comment:
looks like no need to have this if. Can declare `uct_tcp_ep_pending_queue_dispatch` as inline instead

Reply:
done